solidity/deploy/02_deploy_vending_machine.ts

```diff
@@ -0,0 +1,32 @@
+import { HardhatRuntimeEnvironment } from "hardhat/types"
+import { DeployFunction } from "hardhat-deploy/types"
+
+const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
+  const { deployments, getNamedAccounts } = hre
+  const { deploy, log } = deployments
```

Comment:
`log` is not used

Reply:
https://github.com/keep-network/tbtc-v2/pull/17/commits/93dab69fec76b7f22efd5e779795e3d1d669ec8e